### PR TITLE
yara-python 4.3.1 :snowflake:

### PR DESCRIPTION
**Pull request by lorepirri**
yara-python 4.3.1 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-3769]
- dev_url: https://github.com/VirusTotal/yara-python/tree/v4.3.1
- pypi: https://pypi.org/project/yara-python

### Explanation of changes:

- new feedstock, not on conda-forge

### Patches

- `0001-fix-openssl.patch` https://github.com/VirusTotal/yara-python/commit/dff4bf30585a75b233d8dd7e9578082053bce65f
- `0002-fix-enable-openssl.patch` https://github.com/VirusTotal/yara-python/commit/0030658cc7fbb857f2f5a5fbb180344094ff640f
- `0003-bypass-openssl-and-remove-hardcoded-paths.patch` I removed some hardcoded paths, and openssl references
- `0004-enable-optional-modules.patch` I enabled the optional modules using a patch

### Optional Modules

- `magic` dependency on `libmagic` (not on `win`)
- `cuckoo` module, with dependency on `jansson` (which at the moment is not there for `win`, see AnacondaRecipes/jansson-feedstock#2)
- `dex` module, enabled
- `macho` module, enabled

### Depends on

- AnacondaRecipes/jansson-feedstock#2

[PKG-3769]: https://anaconda.atlassian.net/browse/PKG-3769?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PKG-3769]: https://anaconda.atlassian.net/browse/PKG-3769?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by JeanChristopheMorinPerso** [comment]
@lorepirri should this be moved to ready?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - make        # [unix]
+    - patch       # [not win]
+    - m2-patch    # [win]
+  host:
+    - python
+    - setuptools
+    - pip
+    - wheel
+    # jansson 2.14 for win (win-64/jansson-2.14-h2bbff1b_0.tar.bz2) 
+    # has a jansson_d.lib and not jansson.lib, nor jansson.dll
```
Comment (rafaelmartins) [inline recipe/meta.yaml]: this is fine for this pr, but feels like the jansson package on windows is somehow broken. it is not usual to ship only a debug static library

**Review by rafaelmartins — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml]: WOuldn't https://github.com/AnacondaRecipes/jansson-feedstock/pull/2 fix the problem @lorepirri ?

Comment (lorepirri) [reply recipe/meta.yaml]: yes, see dependencies in the description

Comment (rafaelmartins) [reply recipe/meta.yaml]: great, that pr looks nice. thanks for taking care!

**Comment by lorepirri** [comment]
> @lorepirri should this be moved to ready?

@JeanChristopheMorinPerso I was waiting for AnacondaRecipes/jansson-feedstock#2 before proceeding. I've now added staging channels.

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: sure! I've updated this one as well, `jansson` is now included with staging channels, and `yara-python` can have `cuckoo` enabled for `win` as well (depends on `jansson`). Thanks for reviewing!

**Comment by rafaelmartins** [comment]
I checked the last commit, looks all good for me

**Comment by lorepirri** [comment]
> I checked the last commit, looks all good for me

thank you @rafaelmartins !

**Review by JeanChristopheMorinPerso — APPROVED** [review]
Great work @lorepirri!

**Comment by lorepirri** [comment]
> Great work @lorepirri!

thank you @JeanChristopheMorinPerso , I am going to merge this one, thanks also a lot for the help and assistance for this one

**Pull request merged**